portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,12 +25,30 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+   
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String quote="Hello Pranjali!";
-   
-    response.setContentType("text/html;");
+    //Create ArrayList<String> object
```

Comment:
Remove this comment. It is clear from the code that you are creating an ArrayList. Add comments only to explain things that cannot be easily understood from the code.